Clockify.Net/Models/Holiday/DatePeriod.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Clockify.Net.Models.Holiday;
+
+public class DatePeriod {
```

Comment:
Why is it made that way? Does my custom serialization not working? It is defined in `JsonSerializerSettings` in `ClockifyClient`.